src/rpc/methods/state/types.rs

```diff
@@ -79,6 +81,21 @@ pub struct MessageGasCost {
 
 lotus_json_with_self!(MessageGasCost);
 
+impl MessageGasCost {
+    pub fn try_new(message: &Message, apply_ret: &ApplyRet) -> anyhow::Result<Self> {
```

Comment:
nit: I think this can still be called `new`, even though it returns a `Result`, like `File::create`

Reply:
Fixed.